raiden-dapp/src/locales/en.json

```diff
@@ -792,10 +793,12 @@
       },
       "injected-provider": {
         "header": "Injected Provider",
+        "description": "Connect to the provider of your web-browser possibly added with an installed plugin.",
```

Comment:
"Connect with a provider in your web browser"

Reply:
✔️